discovery.md

```diff
@@ -195,12 +195,11 @@ the wire format/API defined by this specification.
 #### Service Attributes
 
 The following sections define the attributes that appear in a Service entity.
-
 ##### id
 
 - Type: `String`
-- Description: A unique identifier for this Service. This value MUST be globally
-  unique. While other metadata about this Service MAY change, this value MUST
+- Description: A unique identifier for this Service. This value MUST be unique.
```

Comment:
I think we need to qualify the scope of the uniqueness. Even just saying withing the scope of the DE might be good enough

Reply:
I agree. I would like to see a concept like the authority and schema version URI defined for the schema registry. As the versioning requirements for discovery are different, it could just be a service URI instead of a service version URI.